lib/vimrunner/rspec.rb

```diff
@@ -46,11 +46,16 @@ def vim
   config.include(Vimrunner::Testing)
 
   # Each example is executed in a separate directory
+  # No trace shall be left in the tmp directory otherwise cygwin won't permit
+  # rmdir => vim is outside the directory at the end
+  # TODO: ensure a cd(pwd) à la RAII
+  pwd = Dir.pwd
   config.around(:each) do |example|
     Dir.mktmpdir do |dir|
       Dir.chdir(dir) do
-        vim.command("cd #{dir}")
+        vim.cd(dir)
```

Comment:
This seems a good idea in general but we might want to tweak the order a little and guard against test failures:

```ruby
config.around(:each) do |example|
  original_dir = Dir.pwd

  Dir.mktmpdir do |dir|
    Dir.chdir(dir) do
      begin
        vim.cd(dir)
        example.run
      ensure
        vim.cd(original_dir)
      end
    end
  end
end
```